index.html

```diff
@@ -1,127 +1,111 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <!-- Description -->
-    <meta name="description" content="" />
-    <title></title>
-    <!-- Links and scripts -->
 
-    <script defer src="./index.js" type="module"></script>
-    <script
-      defer
-      src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/js/bootstrap.bundle.min.js"
-      integrity="sha384-HwwvtgBNo3bZJJLYd8oVXjrBZt8cqVSpeBNS5n7C8IVInixGAoxmnlMuBnhbgrkm"
-      crossorigin="anonymous"
-    ></script>
-  </head>
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <!-- Description -->
+  <meta name="description" content="" />
+  <title></title>
+  <!-- Links and scripts -->
 
-  <body id="homePage">
-    <header></header>
-    <main class="bg-theme-grey">
-      <!-- SECTION 1 -->
+  <script defer src="./index.js" type="module"></script>
```

Comment:
defer have no effect on module scripts. Meaning script tag with type="module" does not need to use defer as they defer by default.

Reply:
done!